python_modules/dagster/dagster_tests/general_tests/grpc_tests/test_persistent.py

```diff
@@ -548,7 +549,46 @@ def test_load_timeout():
     assert "StatusCode.UNAVAILABLE" in str(timeout_exception)
 
 
-def test_load_timeout_code_server_cli():
+def test_server_heartbeat_timeout_code_server_cli() -> None:
```

Comment:
looks like this new test is failing in CI?

Reply:
hmm weird. Passed for me locally.